packages/cozy-mespapiers-lib/src/components/Papers/PapersList.jsx

```diff
@@ -52,6 +52,10 @@ const PapersList = ({ papers, isLast }) => {
       ),
     [actionVariant, client, addMultiSelectionFile, popModal, pushModal]
   )
+  const accountLogin = papers?.list?.[0]?.cozyMetadata?.sourceAccountIdentifier
+  const account = accounts?.find(
+    account => account?.auth?.login === accountLogin
+  )
```

Comment:
Seems like small but pretty significant logic, I think we should name it even more explicitly, and it wouldn't be too far fetched to extract it from the view maybe as a hook

Reply:
je l'ai sorti dans une méthode, en ajoutant des tests au passage